common/src/main/java/com/wynntils/screens/base/widgets/TextInputBoxWidget.java

```diff
@@ -103,7 +103,8 @@ public void renderWidget(PoseStack poseStack, int mouseX, int mouseY, float part
 
         doRenderWidget(
                 poseStack,
-                renderedText,
+                renderedTextDetails.a(),
+                renderedTextDetails.b(),
```

Comment:
Why not use the proper variables? Extract pair.b() and use the named variables.

Reply:
Done, but I would be happy to know why this is better like this. Is this a convention? I never heard it was better to extract named variables before passing it so I'm really curious.

Reply:
(I see now that you already have extracted `renderedTextDetails.a()` to `renderedText`...)

Reply:
Thanks for the explanation ! (renderedText was already extracted not by me)